packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -45,8 +45,13 @@ class CodeSizeBody extends StatefulWidget {
 
 class CodeSizeBodyState extends State<CodeSizeBody>
     with AutoDisposeMixin, SingleTickerProviderStateMixin {
-  static const snapshotTabKey = Key('Code Size Snapshot Tab');
-  static const diffTabKey = Key('Code Size Diff Tab');
+  static const snapshotTabKey = Key('Snapshot Tab');
+  static const diffTabKey = Key('Diff Tab');
+  
+  @visibleForTesting
+  static const clearButtonKey = Key('Clear Button');
+  @visibleForTesting
+  static const dropdownKey = Key('Diff Tree Type Dropdown');
```

Comment:
move these const keys into the widget class `CodeSizeBody`. The CodeSizeBodyState can stay private